src/main/java/core/basesyntax/behaviour/FigureAreaCalculator.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.behaviour;
+
+public interface FigureAreaCalculator {

```

Comment:
```suggestion
public interface AreaCalculator {
```
this interface provides method to get area for some object
in future, we could use it not only for figures